### PR TITLE
🎉 New Source: Holiday API [Low Code CDK]

### LINKED ISSUE #20789: New source: Holiday API
## Tell us about the new connector you’d like to have
Source for the [Holiday API](https://holidayapi.com).

## Describe the context around this new connector
This API is handy for all kinds of business settings, when combined with other internal sources of data. The sync is likely run relatively infrequently, since holidays don't change often. 

This is an API source connector ([docs](https://holidayapi.com/docs)), and likely the "holidays" endpoint is the only relevant one for AirByte.

This service is a freemium one, I'm using it with a free account currently for development, potentially paid if it works out.

## Describe the alternative you are considering or using

Rolling my own using their Python library https://github.com/holidayapi/holidayapi-python

## Are you willing to submit a PR?

Yes.

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Developing new connector for the source: Holiday API*
*Closes #20789*

Resolves:
[🎉 New Source: Holiday API](https://github.com/airbytehq/airbyte/issues/20789)

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## 🚨 User Impact 🚨
- No breaking changes, Just addition of new source
<details><summary><strong>New Connector: Source Holiday API</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [ ] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>
Build results
image

</details>

### CODE REVIEW

**Review by natikgadzhi — COMMENTED** [review]
This needs to be refreshed with new version of how Builder makes manifests. Easiest path forward is probably to copy and paste this manifest into builder, let it do a test run, save to server, and export back — you should get a new version out of it.

File: airbyte-integrations/connectors/source-holiday-api/source_holiday_api/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,271 @@
+version: "0.44.0"
```
[the remark this answers is not in the document]
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-holiday-api/source_holiday_api/manifest.yaml:1]: Builder has much redundancy, So I thought about writing in low-code which is compatible with builder :)
Double advantage, so it is clean in the manifest

**Comment by btkcodedev** [comment]
Built with builder :white_check_mark: 
Fomatted :white_check_mark: 
Comments resolved :white_check_mark: 

`airbyte-ci test` results: Pass
image

**Comment by natikgadzhi** [comment]
Thank you, @btkcodedev! 

Okay, I see what's going on. This pull request is from your fork of the repository, and our CI is clearly not happy about that. 

Can you please push your branch to the airbyte repository, and change this PR to merge from that branch to master, instead of your fork? I.e.

```bash
# Add airbytehq/airbyte as your remote. Ignore this line if you already have it, just make sure you're pushing to it.
git remote add upstream git@github.com:airbytehq/airbyte.git
git push upstream holidayapisource
```

Then, edit this PR and change the source branch to the new branch you've pushed to our repo. AFAIK you should have the permissions to do it already.

This applies not just to holiday, but to all your migration PRs as well.

**Comment by btkcodedev** [comment]
I've pushed in master repo, but it seems I don't have permission to change the source branch of PRs

image

I'll Continue from a new PR

**Comment by btkcodedev** [comment]
Continued in https://github.com/airbytehq/airbyte/pull/36986

**Comment by natikgadzhi** [comment]
Thanks man! 👏🏼

**Pull request closed without merging**